src/App.scss

```diff
@@ -79,3 +119,59 @@ $number-of-channels: 3;
     }
   }
 }
+
+#uploadIframe {
+  width: 100%;
+  max-height: 600px;
+  height: 60vh;
+  box-sizing: content-box;
+}
+
+.sp-mic-live {
```

Comment:
I think we deprecated the `sp` prefix at some point (no big deal)

Reply:
eh. I want to do a lot more tidyups to the CSS (and make them SCSS). Another time.